src/db.ts

```diff
@@ -14,12 +14,16 @@ const uri = process.env.MONGO_URI;
 
 class DbHandler {
     private static instance: DbHandler | null = null;
+
+    private dbName?: string;
```

Comment:
rm name

Reply:
name refers to the database name assigned by mongo (like the name of the exact database)